catroid/src/main/java/org/catrobat/catroid/utils/StatusBarNotificationManager.java

```diff
@@ -331,4 +334,11 @@ private void closeNotificationBar() {
 			sendBroadcast(it);
 		}
 	}
+
+	public static int getUniqueNotificationId() {
+		if (uniqueNotificationIdCounter == null) {
+			uniqueNotificationIdCounter = new AtomicInteger(0);
+		}
```

Comment:
Yes, I know that the method has been changed. Still why and how it will be removed?